content/en/docs/admin/admission-controllers.md

```diff
@@ -316,23 +314,24 @@ In any case, the annotations are provided by the user and are not validated by K
 
 ### Initializers (alpha)
 
+This admission controller is introduced in v1.7.
 The admission controller determines the initializers of a resource based on the existing
 `InitializerConfiguration`s. It sets the pending initializers by modifying the
 metadata of the resource to be created.
-For more information, please check [Dynamic Admission Control](/docs/admin/extensible-admission-controllers/).
+For more information, please check [Dynamic Admission Control](/docs/admin/extensible-admission-controllers.md).
```

Comment:
Update the link to point to `/content/en/docs/` where it just has `/docs/` here and everywhere.